gender_novels/corpus_gen.py

```diff
@@ -467,13 +521,16 @@ def get_author_gender(authors):
         if author_gender == None:
             guesser = gender_guesser.Detector()
```

Comment:
Maybe factor out this whole if statement into a separate function so it can be tested without needing to worry that it's using the wikidata routine (for instance, ten years from now when Mingfei Duan has his own Wikipedia page!)